src/history.js

```diff
@@ -0,0 +1,3 @@
+import createHistory from 'history/createHashHistory';
+
+export default createHistory();
```

Comment:
This file would benefit from a comment explaining its purpose.  It's not clear to me what the `history` module is being used for here / why we need it.